### ISSUE TITLE
Support spring cloud bus push notifications via webhook for bitbucket pull requests

### ISSUE THREAD

**Issue by whboyd** [spring-cloud/spring-cloud-config#506]
In the case of webhook requests coming from bitbucket, spring-cloud-config-monitor currently only looks for requests representing pushes. Pull requests can have great benefits in terms of using a git repository to handle configuration data. For example, they can be used to implement review of configuration changes before merging them into a branch that provides configurations to running apps.

**Comment by Pola93** [comment]
Hi, 
I know that this PR was merged long time ago, but there is probably one minor mistake that slipped through. 
When I attached Bitbucket Cloud webhook on a pull request merge event I noticed that one `if` statement in the `BitbucketPropertyPathNotificationExtractor` class, `extract` method code cannot be fulfilled. 

To be exact when header value for `X-Event-Key` key is equal pullrequest:fulfilled you check in the nested `if` statement whether request body consists of "push" key and if it is instance of Map and if it contains "changes" key with value as a instance of Collection.
Here is the problem, pullrequest:fulfilled event payload does not contain such a key. As it is stated in the documentation right [here](https://confluence.atlassian.com/bitbucket/event-payloads-740262817.html)

Is it possible for to fix that in the near future? Or could I possible prepare little PR for this bug?
